howto/how-to-set-up-a-blaster.md

```diff
@@ -0,0 +1,228 @@
+---
+title: How to set up a blaster
+---
+
+# Introduction
+If you want to setup a blaster, first you have to understand the general mechanics of the `blaster.h` prop. A blaster is always on (unless a dedicated Power button is added). The `blaster.h` prop offers three "modes": Stunt, Kill and Auto. Also, you have a certain number of shots until you need to reload. And you have a certain chance of your weapon randombly jamming.
```

Comment:
This still says "unless a dedicated power button is added" which is somewhat misleading. It's the "poweron*.wav" that controls if there is an on/off function or not.